CorsixTH/Lua/map.lua

```diff
@@ -265,13 +265,45 @@ function Map:load(level, difficulty, level_name, map_file, level_intro, map_edit
     self.parcelTileCounts[plot] = self.th:getParcelTileCount(plot)
     if plot > 1 and not map_editor then
       -- TODO: On multiplayer maps, assign plots 2-N to players 2-N
-      self.th:setPlotOwner(plot, 0)
+      self:setPlotOwner(plot, 0)
     end
   end
 
   return objects
 end
 
+function Map:setPlotOwner(plot_number, new_owner)
+  local split_tiles = self.th:setPlotOwner(plot_number, new_owner)
+  for _, coordinates in ipairs(split_tiles) do
+    local x = coordinates[1]
+    local y = coordinates[2]
+
+    local __, north_wall, west_wall = self.th:getCell(x, y)
```

Comment:
I think luacheck explicitly ignores redefining _ so it can be reused in cases like this. Nicer than introducing a second ignore name.